src/screens/Guild/index.tsx

```diff
@@ -32,20 +50,74 @@ const schema = yup.object().shape({
 });
 
 const Guild = () => {
+  const toast = useToast();
+  const navigation = useNavigation();
+  const isFocused = useIsFocused();
   const { colors } = useTheme();
   const {
     control,
     handleSubmit,
     formState: { errors },
-    setValue,
   } = useForm<FormData>({ resolver: yupResolver(schema) });
   const [isLoading, setIsLoading] = useState<boolean>(true);
+  const [isLoadingRequest, setIsLoadingRequest] = useState<boolean>(false);
   const [hasGuild, setHasGuild] = useState<boolean>(false);
+  const [guild, setGuild] = useState<GuildDto>({} as GuildDto);
   const [guilds, setGuilds] = useState<GuildDto[]>([]);
+  const [image, setImage] = useState<string | undefined>(undefined);
```

Comment:
can we change to null?

Reply:
The react native image component expects a "string | undefined"